deployment/terraform/metrics.go

```diff
@@ -352,6 +353,29 @@ func (t *Terraform) setupMetrics(extAgent *ssh.ExtAgent) error {
 		return fmt.Errorf("error while uploading dashboard_json: output: %s, error: %w", out, err)
 	}
 
+	// Download dashboard v2 from and upload it
```

Comment:
There's something missing here. From grafana.com, you meant?

Reply:
🦅 👁️ 👍